commands/initcmd.go

```diff
@@ -130,7 +130,7 @@ func (io *KptInitOptions) updateKptfile() error {
 		return err
 	}
 	// Finally, set the inventory parameters in the Kptfile and write it.
-	kf.Inventory = &kptfile.Inventory{
+	kf.Inventory = &v1alpha2.Inventory{
```

Comment:
We should use another name when we import this package. This name is not helpful.

Reply:
What do you think is the better name. Should we retain the name `kptfile` ? If so should we change the package name of types.go to `kptfile`?

Reply:
We don't want to change the actual name of any packages, we just want to use an alias when we import this package. So if there won't be any conflicts with the old `kptfile` package, we can just use that as an alias. Otherwise maybe something like `kptfilev1alpha2`. I think the latter would be roughly in line with how we usually handle this in OSS Kubernetes.

Reply:
Done. Added an alias `kptfilev1alpha2 `